ui-library-click-to-call/README.md

```diff
@@ -47,3 +53,1123 @@ Instead, it will copy all the configuration files and the transitive dependencie
+      <CallComposite
+        options={{
+          callControls: {
+            cameraButton: useVideo,
+            screenShareButton: useVideo,
+            moreButton: false,
+            peopleButton: false,
+            displayType: "compact",
+          },
+          localVideoTileOptions: {
+            position: !useVideo ? "hidden" : "floating",
+          },
+        }}
+        adapter={adapter}
+      />
+    </Stack>
+  );
+};
+```
+
+For our CallComposite we have some configuration to do for Click to Call. Depending on your use case we have a number of customizations that can change the user experience. This sample chooses to hide the local video tile, camera, and screen sharing controls if the user opts out of video for their call. Other customizations seen include in the `afterCreate` function defined in the snippet we auto join the call, this will bypass the configuration screen and drop the user into the call with their mic live. Just remove the call to `adapter.join(true)` and the configuration screen will show as normal. Next we will talk about how to get this screen the information once we have our CallComposite configured.
+
+To do this, we will create some handlers to send post messages between the parent window and child window to signal that we want some information. See diagram:
+
+```mermaid
+graph LR
+  child[Child Window]
+  parent[Parent Window]
+  
+
+  parent --> |2. Adapter args| child
+  child -->|1. Args please| parent
+  
+```
+This flow illustrates that if the child window has spawned it needs to ask for the arguments. This has to do with React and that if the parent window just sends a message right after creation the call adapter arguments needed are lost when the application mounts. This is because in the new window the listener is not set yet until after a render pass completes. More on where these event handlers are made to come.
+
+Now we will want to update the splash screen we created earlier. First we will add a reference to the new child window that we will create.
+
+`ClickToCallScreen.tsx`
+
+```typescript
+    ...
+    const [userDisplayName, setUserDisplayName] = useState<string>();
+    const newWindowRef = React.useRef<Window | null>(null);
+    const [useVideo, setUseVideo] = useState<boolean>(false);
+    ...
+```
+
+Next we will create a handler that we pass to our widget that will create a new window that will start the process of sending the post messages.
+
+```typescript
+    ...
+    const startNewWindow = useCallback(() => {
+        const startNewSessionString = 'newSession=true';
+        newWindowRef.current = window.open(
+            window.origin + `/?${startNewSessionString}`,
+            'call screen',
+            'width=500, height=450'
+        );
+    }, []);
+    ...
+```
+
+What this handler does is start a new window position and place a new query arg in the window URL so that the main application knows that it is time to start a new call.
```

Comment:
nit: This could be simpler. This handler starts a new window position and places a ...